spec/features/discover/discover_spec.rb

```diff
@@ -12,28 +12,29 @@
     end
 
     it "should have a button to discover top 40 movies" do
-      visit discover_path
+      VCR.use_cassette("top_40_rated_movies") do
+        visit discover_path
 
-      click_button "Discover Top 40 Movies"
-      expect(current_path).to eq('/movies')
+        click_button "Discover Top 40 Movies"
+        expect(current_path).to eq('/movies')
+      end
     end
 
-    it "should have a search bar for movies" do
-      visit discover_path
+    xit "should have a search bar for movies" do
```

Comment:
We'll implement the search bar functionality on a new branch. Just a reminder that we have some tests skipped for now that relate to the search bar...